quickstarts/microsoft.hardwaresecuritymodules/payment-hsm-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
for any param that does not need a unique value use a defaultValue in the template to make it easier for customers to deploy

Reply:
Added defaults.